inc/templates/pattern-library.php

```diff
@@ -7,12 +7,29 @@
 
 $directories = [];
 
-if ( is_child_theme() ) {
-	$directories[] = get_stylesheet_directory() . '/template-parts';
+$query_var = get_query_var( \Kalutara\Rewrites\QUERY_VAR );
+
+// Build list of template files to display.
+// Either a single template or find all.
+if ( $query_var !== 'all' ) {
+	$templates = [ sanitize_text_field( $query_var ) ];
+} else {
+	$template_files = [];
```

Comment:
I don't see this variable used elsewhere, should it be `$templates`?

Reply:
Fixed